perception/occupancy_grid_map_outlier_filter/src/occupancy_grid_map_outlier_filter_nodelet.cpp

```diff
@@ -230,13 +240,39 @@ void OccupancyGridMapOutlierFilterComponent::onOccupancyGridMapAndPointCloud2(
   stop_watch_ptr_->toc("processing_time", true);
   // Transform to occupancy grid map frame
   PointCloud2 ogm_frame_pc{};
-  if (!transformPointcloud(*input_pc, *tf2_, input_ogm->header.frame_id, ogm_frame_pc)) {
+  PointCloud2 input_infront_pc{};
+  PointCloud2 input_behind_pc{};
+  PointCloud2 ogm_frame_input_behind_pc{};
+  PclPointCloud tmp_behind_pc;
+  PclPointCloud tmp_infront_pc;
+
+  for (sensor_msgs::PointCloud2ConstIterator<float> x(*input_pc, "x"), y(*input_pc, "y"),
+       z(*input_pc, "z");
+       x != x.end(); ++x, ++y, ++z) {
+    if (*x < 0.0) {
+      tmp_behind_pc.push_back(pcl::PointXYZ(*x, *y, *z));
+    } else {
+      tmp_infront_pc.push_back(pcl::PointXYZ(*x, *y, *z));
+    }
+  }
+  pcl::toROSMsg(tmp_infront_pc, input_infront_pc);
+  pcl::toROSMsg(tmp_behind_pc, input_behind_pc);
+  input_infront_pc.header = input_pc->header;
+  input_behind_pc.header = input_pc->header;
```

Comment:
This code would be easier to understand if it were a function.

Reply:
@yukkysaito thank you for your review, I changed to function here [9908ca8](https://github.com/autowarefoundation/autoware.universe/pull/2905/commits/9908ca87301ca9f144d3695eb3a2ab06e79f9a90)